core/workspace_svg.js

```diff
@@ -509,6 +522,9 @@ Blockly.WorkspaceSvg.prototype.translate = function(x, y) {
       'scale(' + this.scale + ')';
   this.svgBlockCanvas_.setAttribute('transform', translation);
   this.svgBubbleCanvas_.setAttribute('transform', translation);
+  if (this.blockDragSurface_) {
+    this.blockDragSurface_.translateAndScaleGroup(x, y, this.scale);
```

Comment:
I believe it basically makes no difference. You could move it to the block drag start, which would add some trivial amount of time there - but should be insignificant compared to the repaint which is about to happen. OTOH, calling it whenever the workspace translates and scales should also be very fast, since the element is hidden and there should be nothing in the SVG. I originally put it here because it made most sense to me conceptually as the goal is to match the workspace translation/scale.